src/OrchardCore/OrchardCore.Search.Elasticsearch.Core/Services/ElasticIndexManager.cs

```diff
@@ -101,8 +101,7 @@ public async Task<bool> CreateIndexAsync(ElasticIndexSettings elasticIndexSettin
             var analyzersDescriptor = new AnalyzersDescriptor();
             var indexSettingsDescriptor = new IndexSettingsDescriptor();
 
-            // The name "standardanalyzer" is a legacy used prior OC 1.6 release. It can be removed in future releases.
-            var analyzerName = (elasticIndexSettings.AnalyzerName == "standardanalyzer" ? null : elasticIndexSettings.AnalyzerName) ?? "standard";
```

Comment:
We don't have a migration for this, nor a note instructing people to migrate [in the release notes](https://docs.orchardcore.net/en/latest/releases/1.6.0/#orchardcoresearchelasticsearch-module). So, this should rather stay, and also `Services/ElasticIndexSettingsService`.

Reply:
I reverted this change